packages/blockchain-wallet-v4-frontend/src/components/SimpleBuy/MultiRowContainer.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components'
+
+const MultiRowContainer = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  width: 230px;
```

Comment:
Why hardcode?

Reply:
for better alignment, will re-check about 100%